cypress-tests/cypress/e2e/configs/Payment/Stripe.js

```diff
@@ -540,6 +540,15 @@ export const connectorDetails = {
       },
       ...commonConnectorDetails.card_pm.MITAutoCapture,
     }),
+    MITWithoutBillingAddress: {
+      Configs: {
+        CONNECTOR_CREDENTIAL: {
+          specName: ["connectorAgnosticNTID"],
+          value: "connector_2",
+        },
```

Comment:
why? i do not see this config being used in ntid test file at all. so, this config can be removed.